### PR TITLE
Components

### PR DESCRIPTION
**Pull request by daudi13**
## Description

Designed the UI of a calculator using class components

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS requirements

- [x] Followed best practices for HTML & CSS
- [x] Followed best practices for JavaScript

### Project requirements

- [x] Delete all the boilerplate from CRA (text, images, styles).
- [x] Create a directory called `components`.
- [x] inside `components` created a new `Calculator.js` file.
- [x] In `Calculator.js`, create a React component that matches this [design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png)
- [x] Don't add any extra design.
- [x] Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- [x] Use class based components.
- [x] Once the Calculator is ready, import it and use it in your main component (App.js).
- [x] Test it in the browser; it should render without problems.
---

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import './style.css';
+
+function Calculator() {
```
Comment (mikethreels) [inline src/components/Calculator.js]: - [x] according to the requirements this component should be class-based please change this before requesting another review.

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App.js';
-import reportWebVitals from './reportWebVitals.js';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (mikethreels) [inline src/index.js]: - [x] the `App` component should call the `Calculator` component and it should be class-based as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi David, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [x] in the public folder there are still some files created by the CRA please remove all files created by the CRA which you don't need.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (daudi13) [reply src/components/Calculator.js]: Function refactored to a class component

File: src/index.js (hunk 2 above)
Comment (daudi13) [reply src/index.js]: Resolved

**Review by rustam-tolipov — APPROVED** [review]
## Hello @daudi13 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**